cli/azd/pkg/apphost/generate.go

```diff
@@ -1042,7 +993,7 @@ func (b *infraGenerator) Compile() error {
 			KeyVaultSecrets: make(map[string]string),
 		}
 
-		ingress, err := buildIngress(docker.Bindings)
+		ingress, err := buildAcaIngress(docker.Bindings, 80)
```

Comment:
Just checking my understanding - while we pass `80` here as a default port, for dockerfiles the manifest is always going to tell us the ingress port so we won't end up using this, but the contract of `buildAcaIngress` forces us to pass something, so we pass the default HTTP port, but we know it won't be used?

Reply:
Yes,  `buildAcaIngress` doesn't know if it is building for a docker file, container or a project.
All it knows is that there might be a case where there are http bindings without explicitly mentioning a targetPort. For that only case, the method is asking what port to use as default. And it is not baked inside the method because we use 80 or 8080